### PR TITLE
change org to build-boiler

### PR DESCRIPTION
**Pull request by dtothefp**
Add hipster script to change org to `build-boiler` 😹

### CODE REVIEW

File: scripts/change-org.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+const glob = require('globby');
```
Comment (stephenkao) [inline scripts/change-org.js]: If you wanted to do more neckbeard and less hipster, you could do something like:

```sh
find . -name '*package.json' -print0 | xargs -0 perl -pi -e 's#github.com/dtothefp/build-boiler/#github.com/build-boiler/build-boiler/#g'
```

:neckbeard: :neckbeard: :neckbeard: :neckbeard:

**Comment by stephenkao** [comment]
This looks good to me!

File: scripts/change-org.js (hunk 1 above)
Comment (dtothefp) [inline scripts/change-org.js]: uggg.... neckbeard buzz kill. where is my "meow" command

Comment (stephenkao) [inline scripts/change-org.js]: 😿

**Pull request merged**